File: Common Modules/CommonTypes.vb
Public Class UserRecord
    Public SIId As Integer
    Public PositionID As Integer
    Public Position As String
    Public FName As String
    Public LName As String
    Public TypeID As Integer
    Public Type As String
    Public InActive As Integer
    Public Address As String
    Public City As String
    Public State As String
    Public Zip As String
    Public HPhone As String
    Public Radio As String
    Public Cell As String
    Public Fax As String
    Public EMail As String
    Public UserID As String
    Public Password As String
    Public SupervisorID As Integer

    ' Backing storage -- a generic dictionary
    Dim stateCaps As New Dictionary(Of String, String)

    ' Add items to the dictionary
    'stateCaps.Add( "WA" , "Olympia")
    'stateCaps.Add( "NM" , "Santa Fe")


End Class
Public Class SecurityMatrix
    Public ObjectName As String
    Public ObjectType As String
    Public EditRank As Integer
    Public ViewRank As Integer
    Public PositionID As Integer
End Class

File: Common Modules/CommonTypes.cs
using System.Collections.Generic;

namespace BossAdmin
{
    public class UserRecord
    {
        public int SIId;
        public int PositionID;
        public string Position;
        public string FName;
        public string LName;
        public int TypeID;
        public string Type;
        public int InActive;
        public string Address;
        public string City;
        public string State;
        public string Zip;
        public string HPhone;
        public string Radio;
        public string Cell;
        public string Fax;
        public string EMail;
        public string UserID;
        public string Password;
        public int SupervisorID;

        // Backing storage -- a generic dictionary
        private Dictionary<string, string> stateCaps = new Dictionary<string, string>();

        // Add items to the dictionary
        // stateCaps.Add( "WA" , "Olympia")
        // stateCaps.Add( "NM" , "Santa Fe")


    }

    public class SecurityMatrix
    {
        public string ObjectName;
        public string ObjectType;
        public int EditRank;
        public int ViewRank;
        public int PositionID;
    }
}